ambry-router/src/test/java/com/github/ambry/router/UndeleteManagerTest.java

```diff
@@ -151,8 +154,7 @@ public void basicThroughRouterTest() throws Exception {
     for (String blobId : blobIds) {
       deleteBlobInAllServer(blobId);
       // Undelete requires global quorum, so we have to make sure all the mock servers received a delete.
-      router.undeleteBlob(blobId, UNDELETE_SERVICE_ID)
-          .get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+      router.undeleteBlob(blobId, UNDELETE_SERVICE_ID).get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.